recipe/meta.yaml

```
@@ -0,0 +1,92 @@
+{% set name = "lightweight-mmm" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name.replace('-', '_') }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3e82b076400cd00eda80be20f23d4a1e3b6192846fbc6d9151d3155f61aeac13
+  patches:
+    - 0001-Increase-matplotlib-version.patch
+    - 0002-JAX-pass-the-condition-x-y.patch
+    - 0003-Disable-plot-show.patch # Set non-interactive backend
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  # Supported versions
+  # https://github.com/google/lightweight_mmm/blob/v0.1.9/setup.py#L74
+  skip: True # [py<310 or py>=311 or s390x]
+  
+
+requirements:
+  build: 
+    - patch # [not win]
```

```suggestion
    - patch  # [not win]
```

two spaces before the `#`